### PR TITLE
Create Dates-and-Reminders.md

### PR DESCRIPTION
**Pull request by joesepi**
WIP: Need to add actual dates.

### CODE REVIEW

**Comment by joesepi** [comment]
Add a reminder to meeting template to check on this doc.

File: Dates-and-Reminders.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Dates and Reminders
+
+This document will be used to capture dates and reminders to review before each CPC meeting. This is a manual process that we hope will be replaced by automated tooling at some point in the near future.
+
+## Recurring dates
+
+- CPC Chair election
+- Director 1 election
```
Comment (joesepi) [inline Dates-and-Reminders.md:8]:
```suggestion
- Director 1 election - term ends February 28
```

File: Dates-and-Reminders.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Director 2 election
```
Comment (joesepi) [inline Dates-and-Reminders.md:9]:
```suggestion
- Director 2 election - term ends August 31
```

File: Dates-and-Reminders.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- CPC Impact voting member selection
```
Comment (joesepi) [inline Dates-and-Reminders.md:10]:
```suggestion
- CPC Impact voting member selection - terms end March 31
```

File: Dates-and-Reminders.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Dates and Reminders
+
+This document will be used to capture dates and reminders to review before each CPC meeting. This is a manual process that we hope will be replaced by automated tooling at some point in the near future.
+
+## Recurring dates
+
+- CPC Chair election
```
Comment (joesepi) [inline Dates-and-Reminders.md:7]: This year, the CPC chair election happened ending on Aug 17. Should we choose a different date to make the official end of term? Perhaps not the same date as the Secondary Director election or is it not a big deal?

File: Dates-and-Reminders.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- Director 1 election - term ends February 28
+- Director 2 election - term ends August 31
+- CPC Impact voting member selection - terms end March 31
+- CPC Non-impact voting member election
```
Comment (joesepi) [inline Dates-and-Reminders.md:11]:
```suggestion
- CPC Non-impact voting members (2) election
```

Comment (joesepi) [reply Dates-and-Reminders.md:11]: Last election concluded May 14

https://github.com/openjs-foundation/cross-project-council/issues/751#issuecomment-841501713

File: Dates-and-Reminders.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (joesepi) [inline Dates-and-Reminders.md:12]:
```suggestion
- Voting members (2) as elected by Regular members
```

Comment (joesepi) [reply Dates-and-Reminders.md:12]: Last election also concluded May 14

https://github.com/openjs-foundation/cross-project-council/issues/752#issuecomment-841502075

**Comment by tobie** [comment]
I'm wondering if using a versioned file for this is really the best solution. What if we maintained an issue instead? Wouldn't that be easier to modify and keep up to date?

**Comment by joesepi** [comment]
> I'm wondering if using a versioned file for this is really the best solution. What if we maintained an issue instead? Wouldn't that be easier to modify and keep up to date? Opened a new issue to showcase what this would look like.

Thanks for looking to optimize @tobie. I really dont know which one would be better -- I'm open to what others think is best.

**Comment by mhdawson** [comment]
I'm good either way.  A versioned file would be relatively easy to maintain if we removed the requirements for review (ie you could commit directly to master)

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by tobie — APPROVED** [review]

**Comment by SaraJo** [comment]
LGTM

**Review by SaraJo — COMMENTED** [review]
LGTM

**Pull request merged**